### ISSUE TITLE
driver.js will make an error if it is used in the scroll area

### ISSUE THREAD

**Issue by MrSeaWave** [kamranahmedse/driver.js#232]
[sourcecode](https://codesandbox.io/s/scroll-driver-enipf)
```
<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="UTF-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
    <title>Static Template</title>
    <link
      rel="stylesheet"
      href="https://unpkg.com/driver.js/dist/driver.min.css"
    />
  </head>
  <style>
    .wrapper {
      background: #cff;
      height: 300px;
      overflow: auto;
    }
    .content {
      background: #ccf;
      height: 1000px;
      display: flex;
      flex-direction: column;
      justify-content: space-between;
    }
  </style>
  <body>
    <div class="wrapper">
      <div class="content">
        <button class="btn-start">start</button>
        <button class="btn-middle">middle</button>
        <button class="btn-end">end</button>
      </div>
    </div>
    <script src="https://unpkg.com/driver.js/dist/driver.min.js"></script>
    <script type="application/javascript">
      const driver = new Driver();
      driver.defineSteps([
        {
          element: ".btn-start",
          popover: {
            title: "start",
            description: "start"
          }
        },
        {
          element: ".btn-middle",
          popover: {
            title: "middle",
            description: "middle"
          }
        },
        {
          element: ".btn-end",
          popover: {
            title: "end",
            description: "end"
          }
        }
      ]);
      driver.start();
    </script>
  </body>
</html>

```

`background` is scroll:

![background](https://user-images.githubusercontent.com/21967852/78325656-d49a6600-75aa-11ea-9e3b-7f38e103b9be.gif)

1.  look at this gif  ,  the middle popover is wrong

![scrollDriverNext](https://user-images.githubusercontent.com/21967852/78325252-b41ddc00-75a9-11ea-85fe-ceac0cf019fc.gif)

2. in the end area, I click the previous, this middle or start is wrong, I can't see the middle popover

![scrollDriverPrevious](https://user-images.githubusercontent.com/21967852/78325566-90a76100-75aa-11ea-95dc-911e0a048e4b.gif)

**Comment by gaocong1990** [comment]
bro, did you  fixed this?

**Comment by kamranahmedse** [comment]
We just released [v1.0](https://github.com/kamranahmedse/driver.js/releases/tag/1.0.3) of driver.js which has been rewritten from the ground up. This issue should no longer exist. Please have a [look at the docs](https://driverjs.com/) and do let me know if you face any issues.
